server/db/src/main/resources/com/walmartlabs/concord/server/db/v1.96.0.xml

```diff
@@ -47,4 +47,40 @@
             </column>
         </addColumn>
     </changeSet>
+
+    <changeSet id="1960012" author="g0h04k0walmart.com">
+        <createTable tableName="SECRET_PROJECTS">
```

Comment:
Maybe `PROJECT_SECRETS`? Otherwise it sounds like projects themselves are secret :-)

Reply:
Agree. :) Renamed to PROJECT_SECRETS .